ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/V2.hs

```diff
@@ -573,13 +572,14 @@ getEnvSTM (LDBHandle varState) f =
 getStateRef ::
   (IOLike m, Traversable t) =>
   LedgerDBEnv m l blk ->
+  ResourceRegistry m ->
```

Comment:
Update the documentation of the function, which currently still says

> The caller is responsible for closing the handle.

Reply:
Done